### PR TITLE
Bump docker/dind version for better CentOS support

### PR DESCRIPTION
**Pull request by RobinKnipe**
Fix errors encountered during `yum` install/update/clean etc.

### CODE REVIEW

**Comment by mhumeSF** [comment]
@RobinKnipe Looking at the previous dind bump, I think tags have to be updated. https://github.com/drone-plugins/drone-docker/pull/161/files

**Comment by tboerger** [comment]
@mrueg this plugin got tags for multiple different docker versions, just pin your builds to the version you want to have. If you are using no tag it's possible that your builds break because we are following more or less the latest versions.

File: docker/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # docker build --rm -f docker/Dockerfile -t plugins/docker .
 
-FROM docker:17.12.0-ce-dind
+FROM docker:18.05.0-ce-dind
```
Comment (bradrydzewski) [inline docker/Dockerfile]: also need to update the `.drone.yml` and bump all versions`18.05`

```diff
-tags: [ "latest", "17", "17.12" ]
+tags: [ "latest", "18", "18.05" ]
```

**Comment by jpds** [comment]
I have a pull request with the requested changes at #194.

**Comment by tboerger** [comment]
We are already publishing 18.06

**Pull request closed without merging**